packages/@expo/metro-config/src/serializer/serializeChunks.ts

```diff
@@ -357,6 +365,20 @@ class Chunk {
 
     const assets: SerialAsset[] = [jsAsset];
 
+    const mutateSourceMapWithDebugId = (sourceMap: string) => {
+      // TODO: Upstream this so we don't have to parse the source map back and forth.
+      if (!debugId) {
+        return sourceMap;
+      }
+      // NOTE: debugId isn't required for inline source maps because the source map is included in the same file, therefore
+      // we don't need to disambiguate between multiple source maps.
+      const sourceMapObject = JSON.parse(sourceMap);
+      sourceMapObject.debugId = debugId;
+      // NOTE: Sentry does this, but bun does not.
+      // sourceMapObject.debug_id = debugId;
```

Comment:
Just confirming only one attribute `debugId` will work. 

Sentry Serializer includes both for legacy reasons, which can be omitted when implementation this feature now.